assets/stylesheets/_components.scss

```diff
@@ -20,6 +20,7 @@
 @import 'account-recovery/reset-code-validation/style';
 @import 'account-recovery/style';
 @import 'auth/style';
+@import 'blocks/action-card/style';
```

Comment:
My bad, this new card should actually be moved to the `client/components` folder since it's a basic UI component that is not connected to state. I realized that some days ago, and even opened https://github.com/Automattic/wp-calypso/pull/23365 to make that clearer in our documentation, but forgot to update the initial requirements in the Trello card 😟.